tidy3d/components/data/monitor_data.py

```diff
@@ -2149,11 +2159,15 @@ def eta(self) -> complex:
         return ETA_0 / np.sqrt(eps_complex)
 
     @staticmethod
-    def propagation_phase(dist: Union[float, None], k: complex) -> complex:
+    def propagation_phase(dist: Union[float, None], k: complex, is_2d_simulation: bool) -> complex:
         """Phase associated with propagation of a distance with a given wavenumber."""
```

Comment:
This comment is not related to your changes, but `propagation_phase` and the docstring seems misleading since this is not just a phase factor. It is an overall normalization factor that contains both phase and amplitude decay due to the propagation. Maybe good to fix?